lookout/style/format/benchmarks/quality_report.py

```diff
@@ -119,25 +99,23 @@ def _convert_files_to_xy(self, parsed_files):
         log.info("VNodes number match to expected %d. ", vnodes_expected_number)
         return _convert_files_to_xy_backup(self, parsed_files)
 
-    def capture_report(func, name):
+    reports = {}
+
+    def capture_reports(func):
         @functools.wraps(func)
-        def wrapped_capture_quality_report(*args, **kwargs):
-            if getattr(report, name) is not None:
-                raise RuntimeError("%s should be called only one time." % func.__name__)
+        def wrapped_capture_quality_reports(*args, **kwargs):
```

Comment:
One day we should remove this function patching at all

Reply:
I also do not like it. 
If we found any way to access comments from lookout-sdk without parsing logs we can easily remove this decorator.